Resources/Lang/en_US.json

```diff
@@ -2709,6 +2713,9 @@
   "SoulCollectorTitle": "SOUL COLLECTOR",
   "CollectOwnSoulOpt": "Can collect their own soul",
   "SoulCollectorSelfVote": "Host settings do not allow you to collect your own soul",
+  "SoulCollectorToDeath": "You have become Death!!!",
+  "SoulCollectorTransform": "<font=\"Brook SDF\"><align=\"center\"><size=150%>Now <b><color=#ff174f>Soul Collector</b></color> has become <b><color=#ff174f>Death</b></color>, Destroyer of Worlds and Horseman of the Apocalypse!</font><font=\"VCR SDF\"><br><br><size=115%><u>Find them and vote them out before it's <color=#ff174f>too late</color>!</u></size></align></font>",
```

Comment:
this works? 
![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/143632280/9220faa1-4454-45ce-9dd2-c5dd4af5225d)
this is when I put it to my template.txt

Reply:
I'm pretty sure you will just have to experiment with invisible text by parsing it in template.txt to then add it as the string.